src/utils/ApiClient.js

```diff
@@ -106,10 +106,46 @@ export default class ApiClient {
     }
 
     /**
-     * Do a PUT call to the Hyperwallet API server
+     * Do a PUT call to the Hyperwallet API server to upload documents
      *
      * @param {string} partialUrl - The api endpoint to call (gets prefixed by `server` and `/rest/v3/`)
      * @param {Object} data - The data to send to the server
+     * @param {api-callback} callback - The callback for this call
+     */
+    doPutMultipart(partialUrl, data, callback) {
+        let contentType = "multipart/form-data";
+        let accept = "application/json";
+        /* eslint-disable no-unused-vars */
+        const keys = Object.keys(data);
+        const values = Object.values(data);
+        /* eslint-enable no-unused-vars */
+
+        let requestDataPromise = new Promise((resolve) => resolve(data));
+        if (this.isEncrypted) {
+            contentType = "multipart/form-data";
+            accept = "application/jose+json";
+            this.createJoseJsonParser();
+            requestDataPromise = this.encryption.encrypt(data);
+        }
+        requestDataPromise.then(() => {
+            request
+                .put(`${this.server}/rest/v3/${partialUrl}`)
+                .auth(this.username, this.password)
+                .set("User-Agent", `Hyperwallet Node SDK v${this.version}`)
+                .type(contentType)
+                .accept(accept)
+                .field(keys[0], JSON.stringify(values[0]))
```

Comment:
what does the data object look like, it seems odd that we are hard coding that the the keys and values. aka, why is it better than doing .filed("foo", data["foo"]). And if the keys are not always the same, are we gauranteed to always have 3 key-value pairs? is it possible to have more than 3 ( do we need to add them programatically)?

Reply:
I tested the API with different combinations (1 file, more than 2 files, more than 1 data object, etc on Insomnia), and it always seems to expect those 3 - data, 2 file attachments, otherwise throws an error.

Reply:
the Data object looks like this: 
   var documents = {
        'documents': [{
            'type': 'DRIVERS_LICENSE',
            'country': 'US',
            'category': 'IDENTIFICATION'
        }]
    };
    var data = {
        "data": documents,
        "drivers_license_front": "/Users/amylavarapu/Desktop/drivers_license_front.jpg",
        "drivers_license_back": "/Users/amylavarapu/Desktop/drivers_license_back.jpg",
    };

Reply:
We have to be careful with this then. because dont think all types of identification/country combinations require 2 photos. unless there is a seperate call for this.

Reply:
I have made it generic, so any number of files can be uploaded. Apparently for letter of authorization, one file is sufficient. Thanks for pointing that out.